docs/source/quickstart.mdx

```diff
@@ -132,7 +248,7 @@ Use the [`~Dataset.to_tf_dataset`] function to set the dataset format to be comp
 
 **7**. Start training with your machine learning framework! Check out the 🤗 Transformers [audio classification guide](https://huggingface.co/docs/transformers/tasks/audio_classification) for an end-to-end example of how to train a model on an audio dataset.
 
-## Vision
+## Image
```

Comment:
The Vision section is missing the "to TF dataset" conversion step - [this](https://github.com/huggingface/transformers/blob/v4.28.1/examples/tensorflow/image-classification/run_image_classification.py#L472C15-L476) example script can help with that.

Reply:
I think the link is not correct, but I've included the `prepare_tf_dataset` call, anyway if we can also include the `to_tf_dataset` alternative that would be nice 😄

Reply:
Now that we have a TF code here, maybe we should use a more framework-agnostic image transforms library (`albumentations` seems to be the best choice)

Reply:
> Now that we have a TF code here, maybe we should use a more framework-agnostic image transforms library (albumentations seems to be the best choice)

Good idea! We have some example code [here](https://huggingface.co/docs/datasets/image_classification) that uses Albumentations that you can use to replace step 2.